app/models/blueprint.rb

```diff
@@ -92,6 +96,10 @@ def publish(bundle_name = nil)
     end
   end
 
+  def archive!
```

Comment:
Do we have a convention for the use of bang methods? From my understanding, it's idiomatic to have a bang method when it is the surprising or dangerous version of its non-bang counterpart.